src/app/ui/modal/common/confirm.component.ts

```diff
@@ -9,8 +9,13 @@ import {Subject} from "rxjs/Subject";
             <div class="body p-1">
                 <span [innerHTML]="content"></span>
             </div>
-            <div class="footer pr-1 pb-1">
-                <button class="btn btn-secondary" (click)="decision.next(false)" type="button">{{ cancellationLabel }}</button>
+            <div class="footer pl-1 pr-1 pb-1">
```

Comment:
Maybe adapt this to `modal-footer` class? It probably didn't exist at the time, but now most modals use it for consistency.